Content/index.md

```diff
@@ -41,6 +41,14 @@ There are some [awesome projects](https://github.com/swiftwasm/awesome-swiftwasm
 
 For more information about Swift and WebAssembly, check out the [Swift and WebAssembly Book](https://book.swiftwasm.org).
 
+### Serverless?
+
+Running and managing your own servers can be complex! Cloud providers offer managed services that take care of most, if not all of the operational burden. This is particularly nice for small systems, because many of these have a free tier.
+
+- [AWS Lambda](https://github.com/swift-server/swift-aws-lambda-runtime)
+- [AWS Lambda Events](https://github.com/swift-server/swift-aws-lambda-events)
+- [Cassandra Driver](https://github.com/apple/swift-cassandra-client)
```

Comment:
Let's move this alongside ElastecSearch and Redis

Reply:
We can add Vercel and Fastly unless @AndrewBarba wants to add them in a separate PR